ecchronos-binary/src/pylib/ecchronoslib/rest.py

```diff
@@ -186,13 +187,19 @@ def list_schedules(self, keyspace=None, table=None):
 
         return result
 
-    def list_repairs(self, keyspace=None, table=None):
+    def list_repairs(self, keyspace=None, table=None, host_id=None):
         request_url = V2RepairSchedulerRequest.v2_repair_status_url
-
         if keyspace and table:
             request_url = "{0}?keyspace={1}&table={2}".format(request_url, keyspace, table)
+            if host_id:
+                request_url += "&hostId={0}".format(host_id)
         elif keyspace:
             request_url = "{0}?keyspace={1}".format(request_url, keyspace)
+            if host_id:
+                request_url += "&hostId={0}".format(host_id)
+        else:
+            if host_id:
+                request_url += "?hostId={0}".format(host_id)
 
```

Comment:
Couldn't all of this be just a single if statement afterwards?

Reply:
nope, if keyspace or table is set we need to append &hostid, if none are specified we need to append ?hostid